src/schemas/json/github-workflow.json

```diff
@@ -841,6 +842,30 @@
                 }
               }
             },
+            "workflow_run": {
+              "$comment": "https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run",
+              "$ref": "#/definitions/eventObject",
+              "description": "This event occurs when a workflow run is requested or completed, and allows you to execute a workflow based on the finished result of another workflow. For example, if your pull_request workflow generates build artifacts, you can create a new workflow that uses workflow_run to analyze the results and add a comment to the original pull request.",
+              "properties": {
+                "types": {
+                  "$ref": "#/definitions/types",
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "requested",
+                      "completed"
+                    ]
+                  },
+                  "default": [
+                    "requested",
+                    "completed"
+                  ]
+                }
```

Comment:
What about adding `workflows`?

An example of workflow YAML:
```yaml
on:
  workflow_run:
    workflows: ["Run Tests"]
    branches: [main]
    types: 
      - completed
      - requested
```

`workflows` is an array of strings.